### PR TITLE
[USM] Support postgres delete operation

### PR DESCRIPTION
**Pull request by amitslavin**
### What does this PR do?

Adds support for DELETE operation in Postgres.

### Motivation

Extend our Postgres support capabilities and increase our accuracy.

### CODE REVIEW

File: pkg/network/protocols/postgres/operations.go
Comment (guyarb) [inline pkg/network/protocols/postgres/operations.go:1]: missing changes in `/pkg/network/encoding/marshal/postgres.go`

Please verify if agent-payload change is required

Comment (amitslavin) [reply pkg/network/protocols/postgres/operations.go:1]: Done and https://github.com/DataDog/agent-payload/blob/906a1ebf65cc83a09fec46025ece3de70b34a7ce/proto/process/agent.proto#L1609

**Review by guyarb — APPROVED** [review]

**Review by guyarb — APPROVED** [review]

**Pull request merged**